boot/assets.go

```diff
@@ -932,6 +936,18 @@ func observeSuccessfulBootAssetsForBootloader(m *Modeenv, root string, opts *boo
 					name:   assetName,
 					hash:   hash,
 				})
+				for _, revokingAsset := range revokingAssets {
```

Comment:
would this a bit cleaner if we iterated over `drop` after outside the main for loop?